readme.txt

```diff
@@ -93,6 +93,22 @@ Would you like to help translate the plugin into more languages? [Join our WP-Tr
 
 == Frequently Asked Questions ==
 
+= Where can I find complete documentation? =
+
+On our web site you can find [our support page](http://docs.easydigitaldownloads.com/?utm_source=docs&utm_medium=faq_tab&utm_content=documentation&utm_campaign=readme)
```

Comment:
This is still not right. It's mentioning the main site and the support page but linking to the documentation.